src/EditorFeatures/Core/Implementation/MetadataAsSource/MetadataAsSourceFileService.cs

```diff
@@ -107,8 +111,10 @@ private string GetRootPathWithGuid_NoLock()
                     var temporaryDocument = _workspace.CurrentSolution.AddProject(temporaryProjectInfoAndDocumentId.Item1)
                                                                      .GetDocument(temporaryProjectInfoAndDocumentId.Item2);
 
-                    var sourceFromMetadataService = temporaryDocument.Project.LanguageServices.GetService<IMetadataAsSourceService>();
-                    temporaryDocument = await sourceFromMetadataService.AddSourceToAsync(temporaryDocument, symbol, cancellationToken).ConfigureAwait(false);
+                    //var sourceFromMetadataService = temporaryDocument.Project.LanguageServices.GetService<IMetadataAsSourceService>();
```

Comment:
:memo: Normally we delete dead code, but this won't be dead code for long. We will need to update this (likely in later pull requests) to do at least the following:

1. Provide an option in Tools &rarr; Options for users to disable decompilation, in which case the old behavior is always used
2. Provide the required legal notice (we are working on getting the specification for this), and if the notice is not accepted then disable decompilation and use the original code
    * As a special case, assemblies marked with `[assembly: SuppressIldasm]` are always excluded from decompilation
3. Add error handling around the decompilation logic, and if decompilation fails then fall back to the original feature